smoketest/scripts/cli/test_interfaces_bonding.py

```diff
@@ -50,6 +51,15 @@ def setUp(self):
         for member in self._members:
             self._options['bond0'].append(f'member interface {member}')
 
+        # Creating test interfaces for port mirroring
+        for mon_intf in self._mirror_interfaces:
+            if 'dum' in mon_intf:
+                self.session.set(['interfaces', 'dummy', mon_intf])
+    
+    def tearDown(self):
+        self.session.delete(['interfaces','dummy','dum10'])
```

Comment:
Why `dum10`, you really need to be more careful!

Reply:
Oh, I haven't tested it yet. I'm just going to 'push' it first, and I'm preparing for the test